lib/core/text.simba

```diff
@@ -696,5 +706,111 @@ begin
 
 end;
 
+(*
+getUpText
+~~~~~~~~~~~~~~~~
+
+.. code-block:: pascal
+
+    function getUpText(): string;
+
+Returns the current UpText (the text in the top-left corner of the screen).
+
+.. note::
+
+    - by slacky & KeepBotting
+    - Last Modified: 27 February 2016 by KeepBotting
+
+Example:
+
+.. code-block:: pascal
+
+    writeln(getUpText());
+*)
+function getUpText(): string;
+var
+  box: TBox := intToBox(4, 5, 500, 25);
+  filter: TCompareRules := [-1, 85, true, 55];
+begin
+  ocr.clientID := exportImageTarget();
+  exit(replaceRegExpr('[^a-zA-Z\d\s]', ocr.recognize(box, filter), '', true));

```

Comment:
Why you choose to remove everything that's not a character or a decimal from the result of `GetUpText`. Would not the real output be more obvious?